### PR TITLE
adds a mentorhelp sound

### PR DESCRIPTION
**Pull request by Amelia0010**
no longer will we be ruled by the honk

### CODE REVIEW

**Comment by Nickvr628** [comment]
what is the sound?

File: yogstation/code/modules/mentor/mentorpm.dm [hunk 1]
```diff
@@ -56,7 +56,7 @@
 	log_mentor("Mentor PM: [key_name(src)]->[key_name(C)]: [msg]")
 
 	msg = emoji_parse(msg)
-	SEND_SOUND(C, sound('sound/items/bikehorn.ogg'))
+	SEND_SOUND(C, sound('yogstation/sound/misc/mentorhelp.ogg'))
```
Comment (Moja10) [inline yogstation/code/modules/mentor/mentorpm.dm:59]: Excellent sound!

**Review by Moja10 — APPROVED** [review]

**Comment by Ktlwjec** [comment]
please make it more space themed

**Comment by thehighwayman** [comment]
my speakers were unable to properly play the high overtones in this. don't merge until nich fixes the balancing in the OGG

**Pull request closed without merging**